core/metacling/src/TClingCallFunc.h

```diff
@@ -77,6 +77,12 @@ class TClingCallFunc {
    bool fReturnIsRecordType : 1;
 
 private:
+   enum EReferenceType {
+      kNotReference,
+      kReference,
```

Comment:
Maybe `kLValueReference`? "Reference" is meant to subsume "r- and l-value reference".